app/javascript/app/components/ndc-sdg/ndc-sdg-linkages-map/ndc-sdg-linkages-map-component.jsx

```diff
@@ -60,31 +61,28 @@ class NdcSdgLinkagesMap extends PureComponent {
     this.setState({ country: country.properties });
 
   renderMapHeader({ isTablet }) {
-    const { downloadLink } = this.props;
+    const { downloadLink, handleInfoClick } = this.props;
     const buttonGroup = (
-      <ButtonGroup
-        className={styles.buttons}
-        buttonsConfig={[
-          {
-            type: 'info',
-            onClick: this.props.handleInfoClick
-          },
-          {
-            type: 'share',
-            shareUrl: '/embed/ndcs-sdg',
-            analyticsGraphName: 'Ndcs-Sdg',
-            positionRight: true
-          },
-          {
-            type: 'download',
-            section: 'ndcs-sdg',
-            link: downloadLink
-          },
-          {
-            type: 'addToUser'
-          }
-        ]}
-      />
+      <React.Fragment>
+        <ButtonGroup
+          className={styles.buttons}
+          buttonsConfig={[
+            {
+              type: 'info',
+              onClick: handleInfoClick
+            },
+            {
+              type: 'download',
+              section: 'ndcs-content',
```

Comment:
I see that you changed the section to 'ndcs-content' for each `ButtonGroup` in this PR - the download link seems to be working but why you changed that exactly?